tests/tests.py

```diff
@@ -149,97 +149,160 @@ def test_rds(self):
-                                                                      u'Name': 'test-dev-internal.kyrtest.pf.dsd.io.',
-                                                                      u'Type': u'A'}]},
-                                     u'Type': u'AWS::Route53::RecordSetGroup'}}
-            ]
+        expected_sgs = {
+            'DefaultSGtestdevexternal': {
+                'Properties': {
+                    u'SecurityGroupIngress': [
+                        {'ToPort': 443,
+                         'IpProtocol': 'tcp',
+                         'CidrIp': '0.0.0.0/0',
+                         'FromPort': 443},
+                        {'ToPort': 80,
+                         'IpProtocol': 'tcp',
+                         'CidrIp': '0.0.0.0/0',
+                         'FromPort': 80}
+                    ],
+                    'VpcId': {'Ref': 'VPC'},
+                    'GroupDescription': 'DefaultELBSecurityGroup'
+                },
+                'Type': u'AWS::EC2::SecurityGroup',
+            },
+            'DefaultSGtestdevinternal': {
+                'Properties': {
+                    u'SecurityGroupIngress': [
+                        {'ToPort': 443,
+                         'IpProtocol': 'tcp',
+                         'CidrIp': '0.0.0.0/0',
+                         'FromPort': 443},
+                        {'ToPort': 80,
+                         'IpProtocol': 'tcp',
+                         'CidrIp': '0.0.0.0/0',
+                         'FromPort': 80}
+                    ],
+                    'VpcId': {'Ref': 'VPC'},
+                    'GroupDescription': 'DefaultELBSecurityGroup'
+                },
+                'Type': 'AWS::EC2::SecurityGroup',
+            }
+        }
 
         config = ConfigParser(
             ProjectConfig(
                 'tests/sample-project.yaml',
                 'dev').config,'my-stack-name')
         elb_cfg, elb_sgs = config.elb()
-        compare(known, elb_cfg)
 
-    def test_elb_no_sgs(self):
-        from testfixtures import compare
+        compare(expected_resources, elb_cfg)
+
+        compare(expected_sgs, elb_sgs)
+
+    def test_elb_custom_sg(self):
+
+        expected_sgs = {
+            'SGName': {
+                'Properties': {
+                    u'SecurityGroupIngress': [
+                        {'ToPort': 443,
+                         'IpProtocol': 'tcp',
+                         'CidrIp': '1.2.3.4/32',
+                         'FromPort': 443},
+                    ],
+                    'VpcId': {'Ref': 'VPC'},
+                    'GroupDescription': 'DefaultELBSecurityGroup'
+                },
+                'Type': u'AWS::EC2::SecurityGroup',
+            },
+        }
+
+        project_config = ProjectConfig('tests/sample-project.yaml', 'dev')
+
+        # Remove the "test-dev-internal" ELB
+        project_config.config['elb'].pop()
```

Comment:
This feels likely to break. Can we check it's name? Or pop the elb key and write your own in the test?